### PR TITLE
Getty and ssh improvements

### PR DESCRIPTION
**Pull request by ijc**
**- What I did**

Updated the `sshd` and `getty` containers such that the following runs ok:
```
dist pull docker.io/library/redis:alpine
ctr run -t docker.io/library/redis:alpine test
```
and produces the usual redis start of day logging.

**- How I did it**

* Added `/etc/localtime` (configured for UTC) to host fs, since `ctr` wants to bind it.
* Added `ca-certificates` to both containers, since `dist` wants some roots
* Mounted host `/tmp` into the containers, `ctr` and `containerd` need to share `/tmp`.
* Synchronised `sshd` containers bind mounts to include all those used by `getty` (which includes the binds of `ctr` and `dist` themselves)

**- How to verify it**

```
dist pull docker.io/library/redis:alpine
ctr run -t docker.io/library/redis:alpine test
```
In either container ought to work

**- Description for the changelog**
Tools such as `ctr` and `dist` now work in both the `getty` and `sshd` containers.

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by ijc** [comment]
CI issue seems to be:
```
+[STDERR  ] 2017-06-23T11:14:02.113547538Z: FAILURE: rpc_pipefs filesystem missing
+[STDERR  ] 2017-06-23T11:14:02.113555812Z: FAILURE: nfs filesystem missing
+[STDERR  ] 2017-06-23T11:14:02.113558701Z: FAILURE: nfs4 filesystem missing
+[STDERR  ] 2017-06-23T11:14:02.113576522Z: FAILURE: nfsd filesystem missing
+[STDERR  ] 2017-06-23T11:14:02.113585612Z: FAILURE: ntfs filesystem missing
+[STDERR  ] 2017-06-23T11:14:02.113591561Z: kernel config test failed!
+[STDERR  ] 2017-06-23T11:14:02.113599855Z: Kernel config test suite FAILED
```

Not sure how that can be the fault of this PR.

**Comment by tych0** [comment]
+1, fixes ctr exec for me:

```
/ # ctr exec -t nginx sh
/ # ps auxf
PID   USER     TIME   COMMAND
    1 root       0:00 nginx: master process nginx -g daemon off;
    5 nginx      0:00 nginx: worker process
   21 root       0:00 sh
   25 root       0:00 ps auxf
```

I think the CI is busted, it's being angry on a few other PRs too.

**Comment by ijc** [comment]
> I think the CI is busted, it's being angry on a few other PRs too.

I think the kernels have been update out of band to have modules somehow, cc @rn.

**Comment by rn** [comment]
Yes, the kernels where pushed but need an update to the kernel configure test. Might be worth merging my kernel update PR as it only failed on the EFI test IIRC

**Comment by ijc** [comment]
Do we have a plan to avoid future breakage caused by changes which have not yet passed CI going live?

**Comment by rn** [comment]
Yes, the idea would be that CI, on PRs would do all builds necessary but push to temporary registry and only push to hub on merge. This is quite handwavey for now as it needs to be thought through properly...

**Comment by justincormack** [comment]
Can you rebase, should fix CI

**Comment by justincormack** [comment]
Also this will conflict with #2098 `init` changes

**Comment by ijc** [comment]
Rebasing now, as discussed IRL I'll move the addition of `/etc/localtime` from `pkg/init` to `pkg/containerd` which is more correct (although still not ideal) but will also avoid the conflict with #2098.

**Comment by ijc** [comment]
Rebase done.

File: tools/alpine/packages [hunk 1]
```diff
@@ -66,6 +66,7 @@ squashfs-tools
 strace
 syslinux
 tar
+tzdata
```
Comment (justincormack) [inline tools/alpine/packages]: thats not actually alphabetical!

Comment (ijc) [inline tools/alpine/packages]: Bugger. Will sort it.

**Pull request merged**